tests/python_functional/functional_tests/template_functions/slog/test_secure_logging.py

```diff
@@ -41,7 +42,8 @@ def test_secure_logging(config, syslog_ng, slog):
     syslog_ng.start(config)
 
     logs = file_destination.read_logs(num_of_messages)
-    assert not any(map(lambda x: example_message in x, logs))
+    # test for no clear text
+    assert not any(map(lambda x: "example_message" in x, logs))
```

Comment:
The "example_message" as a string in assertion does not match with the original input string which is: "example-message". Maybe it would be better to make an own variable for input test messag, as it will be referenced in final assert also

Reply:
Fixed thanks. I introduced a variable: `message_base`.